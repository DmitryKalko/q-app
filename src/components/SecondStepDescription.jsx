import React from "react";
import secondStep from "./svg/second-step.svg";

const SecondStepDescription = ({ lightTheme }) => {
  return (
    <div className="step">
      <p style={lightTheme ? { color: "white" } : { color: "#6d6d6d" }}>
        Please fill in additional information
      </p>
      <img src={secondStep} alt="firstStep" />
    </div>
  );
};
export default SecondStepDescription;
