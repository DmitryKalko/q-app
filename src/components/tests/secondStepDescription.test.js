import { shallow } from "enzyme";
import React from "react";
import SecondStepDescription from "../SecondStepDescription";

describe('about work SecondStepDescription component', () => {
    it('should render SecondStepDescription component', () => {
        const component = shallow(<SecondStepDescription/>)
        expect(component).toMatchSnapshot();
    });
});