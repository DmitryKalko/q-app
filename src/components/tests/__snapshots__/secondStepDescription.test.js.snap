// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`about work SecondStepDescription component should render SecondStepDescription component 1`] = `
<div
  className="step"
>
  <p
    style={
      Object {
        "color": "#6d6d6d",
      }
    }
  >
    Please fill in additional information
  </p>
  <img
    alt="firstStep"
    src={
      Object {
        "process": [Function],
      }
    }
  />
</div>
`;
